application/lib/task1/start.js

```diff
@@ -1,13 +1,13 @@
 async () => {
   if (application.worker.id === 'W1') {
```

Comment:
```suggestion
  if (application) return; // comment to enable
  if (application.worker.id === 'W1') {
```

Reply:
I'd prefer this style to enable/disable such examples

Reply:
You can add this in other `start` handlers

Reply:
I think disable such features are also not a subject of this PR and added just to make all project runable